docs-next/src/content/docs/explainers/nodejs-native-esm-support.mdx

```diff
@@ -29,5 +29,8 @@ More information can be found in the [Node.js documentation](https://nodejs.org/
 - [Watch mode](../running/cli#--watch--w) does not support ES Module test files
 - [Custom reporters](../reporters/third-party) and [custom interfaces](../interfaces/third-party) can only be CommonJS files
 - [Configuration file](../running/configuring) can only be a CommonJS file (`.mocharc.js` or `.mocharc.cjs`)
+- There was a [bug](https://github.com/nodejs/node/issues/58945) in node which caused top level errors in ESM files to be ignored. 
+  The fix landed in node 24.4.1, but if your tests seem to fail silently or dont seem to be detected you can add `--no-experimental-require-module`
+  to the `NODE_OPTIONS` environment variable.
```

Comment:
> [bug](https://github.com/nodejs/node/issues/58945) 

[Accessibility] https://www.w3.org/WAI/WCAG21/Techniques/general/G91 - links should be descriptive in just their text. Some users use link texts to navigate through the page.

Applying a bit of wordcrafting, how does this look to you?

```suggestion
- Mocha in Node.js versions prior to 24.1.1 [silently ignored top level errors in ESM files](https://github.com/mochajs/mocha/issues/5396).
  If you cannot upgrade to that Node.js version or newer, you can add `--no-experimental-require-module`  to the `NODE_OPTIONS` environment variable.
```

Reply:
Thanks will adopt your version, I like framing the version in the past, `versions prior to` but do you mean 24.1.1 literally? 24.4.0 is out and still has the bug :grimacing:

Reply:
I don't personally have a preference, what you changed it to makes sense to me. I suppose we'll know for sure what the range is once they release the next version?